src/app/pages/apps/components/chart-wizard/chart-wizard.component.ts

```diff
@@ -116,6 +116,8 @@ export class ChartWizardComponent implements OnInit, AfterViewInit, OnDestroy {
           nextElement = document.getElementById(id);
           if (idx === path.length - 1) {
             nextElement?.scrollIntoView({ block: 'center' });
+            nextElement.classList.add('highlighted');
+            setTimeout(() => nextElement.classList.remove('highlighted'), 999);
```

Comment:
Use `timer` or `interval` from rxjs as they are cleaner. Or, handle removing the timer when done with it.